Makefile

```diff
@@ -31,9 +31,8 @@ build: ## Build the Karpenter KWOK controller images using ko build
 	$(eval IMG_DIGEST=$(shell echo $(CONTROLLER_IMG) | cut -d "@" -f 2))
 	
 apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the current state of your git repository into your ~/.kube/config cluster
-	hack/validation/kwok-requirements.sh
-	kubectl apply -f pkg/apis/crds
-	helm upgrade --install karpenter kwok/charts --namespace $(KARPENTER_NAMESPACE) --skip-crds \
+	kubectl apply -f kwok/charts/crds
```

Comment:
Why are we applying both a kubectl apply on the chart CRDs and a kubectl apply on kwok API CRDs

Reply:
Are you asking why they're not in the same directory? Or are you wondering why we care about doing two separate CRD applys?

Reply:
I'm trying to understand why we are doing two separate CRD applies if we have the CRDs in the charts directory

Reply:
Closed after adding a symlink to the charts directory to pull in the CRD